_includes/story.md

```diff
@@ -690,7 +699,7 @@ module Workerholic
 end
 ```
 
-Next, we start the ASP algorithm inside a thread so it doesn't block the main thread of execution. The ASP algorithm will be run every second:
+Next, we start the ASP rebalancing inside a thread so it doesn't block the main thread of execution. The ASP rebalancing algorithm will be run every second:
```

Comment:
`rebalancing` adds another keyword to the algorithm which is basically the same as the `P` (provisioning) in `ASP`. Using `ASP algorithm` is fine, the sentence could be rephrased in a better way maybe:

> Next, we use our ASP algorithm inside a thread so it doesn't block the main thread of execution. The ASP algorithm will be run every second: